telluric/georaster.py

```diff
@@ -999,8 +999,8 @@ def pixel_crop(self, bounds, xsize=None, ysize=None, window=None):
         else:
             window = window or rasterio.windows.Window(bounds[0],
                                                        bounds[1],
-                                                       bounds[2] - bounds[0] + 1,
-                                                       bounds[3] - bounds[1] + 1)
+                                                       bounds[2] - bounds[0],
```

Comment:
@guydou proposed

```
max(bounds[2] - bounds[0], 1)
```

but it would be better to first discuss what are we trying to avoid.